apps/dialtone-documentation/docs/design/colors/chart-colors/index.md

```diff
@@ -0,0 +1,248 @@
+Ensuring accessible data visualizations is a core tenet of Dialtone. When using these tokens:
+
+- Contrast Ratios: All token color combinations are designed to meet WCAG AA contrast standards (minimum 3:1 for graphics/UI components).
+- Color-Blindness Compatibility: Palettes have been vetted for common forms of color blindness (e.g., using tools like Viz Palette).
+- Redundancy is Key: Never rely solely on color to convey critical information. Always provide supplementary visual cues such as:
+  - Labels/Text: Directly annotating data points with values.
+  - Icons/Patterns: Using distinct icons or patterns for categories.
+  - Shapes/Stroke Styles: Varying shapes, line styles (dashed, dotted), or stroke widths.
+- Theming Impact: Token values are specifically defined for both light and dark modes to ensure optimal contrast and readability in any theme.
+
+## Usage for Designers
+
+Designers should leverage these tokens directly  from the Dialtone Figma Library.
+
+- Applying Colors: Apply color directly to chart elements (bars, lines, fills) via Figma Styles linked to tokens. Unlike other Dialtone colors, Chart Colors are not scoped to foreground, surface, border, etc.
+- Understanding Hover/Selected: Remember that hover and selected states for specific chart series (single color, semantic, categorical, and sequential) have predetermined contrasting colors, ensuring consistent interaction feedback.
+- Prototyping: Use these tokens in your Figma prototypes to demonstrate interactive states and theme switching.
+
+## Known Issues & Troubleshooting
+
+While this system aims for robustness, be aware of potential challenges during implementation and adoption:
+
+- Charting Library Overrides: Some charting libraries might aggressively override inline styles or custom properties, requiring careful mapping or deeper customization.
+- Complex Color Logic: For very custom data-driven color calculations beyond simple references (e.g., dynamic gradients based on data range), you might still need to implement logic in code that consumes these Chart Color design tokens.
+- Transitioning Existing Charts: Migrating older charts not using design tokens will require dedicated refactoring effort.
+
+## Do and Don'ts
+
+### Use the colors in the right order
+
+<dialtone-usage>
+<template #do>
+ <div class="d-bar8"> <svg-loader class="d-fl1" name="chart-dondont-1-1" />
+ </div>
+Use categorical and sequential colors in their predetermined order, e.g. 01, 02, 03, etc.
+</template>
+
+<template #dont>
+ <div class="d-bar8"> <svg-loader class="d-fl1" name="chart-dondont-1-2" />
+ </div>
+Avoid using categorical and sequential colors out of order, e.g. 10, 03, 05, etc.
+
+</template>
+
+</dialtone-usage>
+
+### Limit the number of colors used in one graph to 6 max
+
+<dialtone-usage>
+<template #do>
+ <div class="d-bar8"> <svg-loader class="d-fl1" name="chart-dondont-2-1" />
+ </div>
+When displaying information, limit the number of colors used in one graph to 6 max.
```

Comment:
```suggestion
Where possible, limit the number of colors in a single graph to no more than 6. 
```
There may be acceptable cases where a chart has to have more than 6, so this at least encourages designers to be judicious.